PROTO_tests/tests/dtypes_test.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+from arkouda import dtypes
+
+"""
+DtypesTest encapsulates arkouda dtypes module methods
+"""
+
+SUPPORTED_NP_DTYPES = [
+    bool,
+    int,
+    float,
+    str,
+    np.bool_,
+    np.int64,
+    np.float64,
+    np.uint8,
+    np.uint64,
+    np.str_,
+]
+
+
+class TestDTypes:
+    @pytest.mark.parametrize("dtype", SUPPORTED_NP_DTYPES)
+    def test_check_np_dtype(self, dtype):
+        dtypes.check_np_dtype(np.dtype(dtype))
+
+    def test_check_np_dtype_errors(self):
+        with pytest.raises(TypeError):
+            dtypes.check_np_dtype(np.dtype(np.int16))
+        with pytest.raises(TypeError):
+            dtypes.check_np_dtype("np.str")
+        with pytest.raises(TypeError):
+            dtypes.check_np_dtype("bigint")
```

Comment:
so doing `"bigint"` is no different than doing `"np.str"` since they are both strings. I think it would be better to updated this to 
```python
dtypes.check_np_dtype(ak.bigint)
```

Reply:
I'm sorry if I wasn't clear before I think we should keep the `"np.str"` check and just edit the `"bigint"` check to do `ak.bigint` instead

Reply:
Nah that was my mistake.  Updated.